HelpfulItem.h

```diff
@@ -10,6 +10,5 @@
 struct HelpfulItem : Item
```

Comment:
Confirm if you need all of these header files before you include them
- [ ] revise